src/Dfc.CourseDirectory.Core/DataStore/Sql/Queries/GetProviderDashboardCounts.cs

```diff
@@ -1,9 +1,13 @@
 ﻿using System;
+using System.Collections.Generic;
+using Dfc.CourseDirectory.Core.Models;
 
 namespace Dfc.CourseDirectory.Core.DataStore.Sql.Queries
 {
-    public class GetProviderDashboardCounts : ISqlQuery<(int CourseRunCount, int ApprenticeshipCount, int VenueCount)>
+    public class GetProviderDashboardCounts : ISqlQuery<(IReadOnlyDictionary<CourseStatus, int> CourseRunCounts, IReadOnlyDictionary<ApprenticeshipStatus, int> ApprenticeshipCounts, int VenueCount, int PastStartDateCourseRunCount, int BulkUploadCoursesErrorCount, int BulkUploadCourseRunsErrorCount, int ApprenticeshipsBulkUploadErrorCount)>
```

Comment:
Given the size of this tuple, maybe a `DashboardCounts` class would be better here?

Reply:
I was thinking that myself!